api/client/samples/similar_regions/region_properties.py

```diff
@@ -9,6 +9,7 @@
             'source_id': 43
         },
         "properties": {
+            "type": "timeseries",
```

Comment:
is that information contained in `frequency_id`?

Reply:
inherently yes - but want to use "type" to denote how to handle the series. so some time series we probably want to do what i do now (fourier transform), but other time series will need different transforms.

I will rename this from "timeseries".

Reply:
rename done.